### ISSUE TITLE
[ERR] Unhanled exception in logfile by nework disconnect (0.11.0)

### ISSUE THREAD

**Issue by golemUser1** [golemfactory/clay#2034]
Currently i have some problems with my WLAN (not your fault), so tested unintentionally how the system reacts to disconnects.
the entries in the logfile look improvable:
2018-02-03 14:55:27 WARNING  golem.monitor.transport             b'Problem sending payload to: 'https://stats.golem.network/''
Traceback (most recent call last):
  File "C:\BuildbotWorker\buildpackage_windows\build\.venv\lib\site-packages\requests\packages\urllib3\connection.py", line 141, in _new_conn
  File "C:\BuildbotWorker\buildpackage_windows\build\.venv\lib\site-packages\requests\packages\urllib3\util\connection.py", line 83, in create_connection
  File "C:\BuildbotWorker\buildpackage_windows\build\.venv\lib\site-packages\requests\packages\urllib3\util\connection.py", line 73, in create_connection
OSError: [WinError 10065] Der Host war bei einem Socketvorgang nicht erreichbar

i had two disconnects in a row. after the first one the golem was not able to reconnect. but managed reconnection after the second incident.

see full logs:

[golem.error.log](https://github.com/golemfactory/golem/files/1691927/golem.error.log)
[golem.log](https://github.com/golemfactory/golem/files/1691928/golem.log)
[hyperg.log](https://github.com/golemfactory/golem/files/1691929/hyperg.log)

**Comment by maaktweluit** [comment]
Hey @golemUser1 !

Thank you for the report!
This is not an error but a warning, I think this issue should be marked UX.

This warning indicates that you disconnected from the monitor at stats.golem.network.
The message could be more clear to indicate this, golem should work regardless of this warning.

In the scenario you describe there should be other errors indicating there is no connection at all.
Looking at your logs these are:
Payment-processor:
```
2018-02-03 14:55:50 ERROR    golem                               b'Service Error: Traceback (most recent call last):\n  File "C:\\Python36\\lib\\threading.py", line 916, in _bootstrap_inner\n    \n  File "C:\\Python36\\lib\\threading.py", line 864, in run\n    \n  File "C:\\BuildbotWorker\\buildpackage_windows\\build\\.venv\\lib\\site-packages\\twisted\\_threads\\_threadworker.py", line 46, in work\n    \n  File "C:\\BuildbotWorker\\buildpackage_windows\\build\\.venv\\lib\\site-packages\\twisted\\_threads\\_team.py", line 190, in doWork\n    \n--- <exception caught here> ---\n  File "C:\\BuildbotWorker\\buildpackage_windows\\build\\.venv\\lib\\site-packages\\twisted\\python\\threadpool.py", line 250, in inContext\n    \n  File "C:\\BuildbotWorker\\buildpackage_windows\\build\\.venv\\lib\\site-packages\\twisted\\python\\threadpool.py", line 266, in <lambda>\n    \n  File "C:\\BuildbotWorker\\buildpackage_windows\\build\\.venv\\lib\\site-packages\\twisted\\python\\context.py", line 122, in callWithContext\n    \n  File "C:\\BuildbotWorker\\buildpackage_windows\\build\\.venv\\lib\\site-packages\\twisted\\python\\context.py", line 85, in callWithContext\n    \n  File "C:\\BuildbotWorker\\buildpackage_windows\\build\\golem\\ethereum\\paymentprocessor.py", line 412, in _run\n    \n  File "C:\\BuildbotWorker\\buildpackage_windows\\build\\golem\\ethereum\\paymentprocessor.py", line 109, in is_synchronized\n    \n  File "C:\\BuildbotWorker\\buildpackage_windows\\build\\golem\\ethereum\\paymentprocessor.py", line 92, in check\n    \n  File "C:\\BuildbotWorker\\buildpackage_windows\\build\\golem\\ethereum\\client.py", line 41, in get_peer_count\n    \n  File "C:\\BuildbotWorker\\buildpackage_windows\\build\\.venv\\lib\\site-packages\\web3\\utils\\functional.py", line 14, in inner\n    \n  File "C:\\BuildbotWorker\\buildpackage_windows\\build\\.venv\\lib\\site-packages\\web3\\net.py", line 24, in peerCount\n    \n  File "C:\\BuildbotWorker\\buildpackage_windows\\build\\.venv\\lib\\site-packages\\web3\\providers\\manager.py", line 44, in request_blocking\n    \n  File "C:\\BuildbotWorker\\buildpackage_windows\\build\\.venv\\lib\\site-packages\\web3\\providers\\rpc.py", line 52, in make_request\n    \n  File "C:\\BuildbotWorker\\buildpackage_windows\\build\\.venv\\lib\\site-packages\\web3\\utils\\compat\\compat_requests.py", line 21, in make_post_request\n    \n  File "C:\\BuildbotWorker\\buildpackage_windows\\build\\.venv\\lib\\site-packages\\requests\\sessions.py", line 535, in post\n    \n  File "C:\\BuildbotWorker\\buildpackage_windows\\build\\.venv\\lib\\site-packages\\requests\\sessions.py", line 488, in request\n    \n  File "C:\\BuildbotWorker\\buildpackage_windows\\build\\.venv\\lib\\site-packages\\requests\\sessions.py", line 609, in send\n    \n  File "C:\\BuildbotWorker\\buildpackage_windows\\build\\.venv\\lib\\site-packages\\requests\\adapters.py", line 487, in send\n    \nrequests.exceptions.ConnectionError: HTTPConnectionPool(host=\'94.23.57.58\', port=55555): Max retries exceeded with url: / (Caused by NewConnectionError(\'<requests.packages.urllib3.connection.HTTPConnection object at 0x000002789B1B8898>: Failed to establish a new connection: [WinError 10065] Der Host war bei einem Socketvorgang nicht erreichbar\',))\n'
```
Resource-manager:
```
2018-02-03 16:03:31 ERROR    golem.resources                     b'Resource handshake error ('6620431515d236653c02e9d95ef480162e1b3c3ad05b35251751d247f5b085c777b468940cbe8aba5244b29369bfc37983a0dc7e57deaeb6ad2817420527fd88'): <twisted.python.failure.Failure requests.exceptions.HTTPError: [Failure instance: Traceback (failure with no frames): <class 'twisted.web._newclient.ResponseNeverReceived'>: [<twisted.python.failure.Failure twisted.internet.error.ConnectionDone: Connection was closed cleanly: Connection done.>]
```

There should also be one for task-related connections.

**Comment by badb** [comment]
@maaktweluit Do you have any suggestions how can we improve the logs to be more readible?

**Comment by maaktweluit** [comment]
I would propose to reduce the level of the complicated messages with UUID's and stack traces to debug level and replace them with ELI5 info messages

**Comment by badb** [comment]
Ok. So we may assume that it's part of this issu: https://github.com/golemfactory/golem/issues/1598
